### DISCUSSION TITLE
How to create a new page with a query on a collection with some filters and export in Excel/CSV?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#11003] [Archive]
How to create a new page with a query on a collection with some filters and export in Excel/CSV?

**Comment by rijkvanzanten** [comment]
You can add the `export` parameter to any API endpoint, so something like

```
/items/articles?fields=*.*&export=csv
```

will do the trick 👍🏻

**Comment by joselcvarela** [comment]
This is how you can set a bookmark and use the export feature with the current filter applied.

https://user-images.githubusercontent.com/14039341/149877327-d229adfe-1108-42f6-b0e6-74fcf677a91d.mov

A bookmark is a saved view with the configurations you set, like filters, sort, limit, etc.
The export on sidebar allows you to export items in CSV or JSON. You can apply the configurations like filters, sort, limit, etc. for the export, or you just can fetch all items without any configurations by setting/unsetting the checkbox, respectively.
